Sources/web3swift/Utils/EIP/EIP712.swift

```diff
@@ -100,7 +100,7 @@ fileprivate extension EIP712Hashable {
     }
 
     func encodePrimaryType() -> String {
-        let parametrs: [String] = Mirror(reflecting: self).children.compactMap { key, value in
```

Comment:
Please undo this changes, as they affecting code, rather then comments and should be reviewed separately.